website/docs/r/iam_service_linked_role.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "aws"
+page_title: "AWS: aws_iam_service_linked_role"
+sidebar_current: "docs-aws-resource-iam-service-linked-role"
+description: |-
+  Provides an IAM service-linked role.
+---
+
+# aws_iam_service_linked_role
+
+Provides an [IAM service-linked role](https://docs.aws.amazon.com/IAM/latest/UserGuide/using-service-linked-roles.html).
+
+## Example Usage
+
+```hcl
+resource "aws_iam_service_linked_role" "elasticbeanstalk" {
+  aws_service_name = "elasticbeanstalk.amazonaws.com"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `aws_service_name` - (Required, Forces new resource) The AWS service to which this role is attached. You use a string similar to a URL but without the `http://` in front. For example: `elasticbeanstalk.amazonaws.com`
```

Comment:
We should probably link to documentation about where to find these names here. 😄

Reply:
I couldn't seem to find a list of all of the service domains, but the closest thing I found was [this](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_aws-services-that-work-with-iam.html). Do you think it's sufficient?